### PR TITLE
Update dependencies and add NPM lockfile

### PR DESCRIPTION
**Pull request by qwerty287**
- update PHP dependencies (`composer update`)
- add an NPM lockfile to lock JS dependencies for Livewire.

### CODE REVIEW

File: app/Metadata/Extractor.php [hunk 1]
```diff
@@ -341,14 +341,14 @@ public static function createFromFile(NativeLocalFile $file): self
 							$taken_at->setTimezone(new \DateTimeZone(date_default_timezone_get()));
 						}
 						// In the remaining cases the timezone information was
-						// extracted and the recording time is assumed exhibit
-						// to original timezone of the location where the video
-						// has been recorded.
-						// So we don't need to do anything.
+					// extracted and the recording time is assumed exhibit
```
Comment (ildyria) [inline app/Metadata/Extractor.php]: weird change.

Comment (qwerty287) [reply app/Metadata/Extractor.php]: This was php-cs-fixer, let me look at it.

Comment (qwerty287) [reply app/Metadata/Extractor.php]: Hmm yes that's weird. Actually, I downgraded php-cs-fixer again after I noticed that it doesn't work... I'll try to find out why it'd doing this.

**Comment by qwerty287** [comment]
Everything's fixed again.

**Comment by ildyria** [comment]
Honestly, I would not bother much with that PR. The reason being that #1403 should allow user to use [Laragear/WebAuthn](https://github.com/Laragear/WebAuthn) instead of [our Larapass fork](https://github.com/LycheeOrg/Larapass).
By doing so we will be migrating to Laravel v9 and this will apply a whole bunch of package update on its own.

**Comment by qwerty287** [comment]
@ildyria  It's finished, I fixed everything :)

**Review by ildyria — APPROVED** [review]

**Comment by nagmat84** [comment]
I did not know that we also use NPM for the Lychee backend; the the frontend of course, yes. Could anybody enlighten me why we need NPM and by what it is used?

**Comment by qwerty287** [comment]
It will be used for Livewire.

**Comment by ildyria** [comment]
NPM is used to compile dependencies from the `resources/asset` folder.

**Comment by ildyria** [comment]
> It will be used for Livewire.

Exactly that. :laughing:

**Pull request merged**